macros/calibration/pa_calibrate.cfg

```diff
@@ -132,6 +132,7 @@ gcode:
       "Your final Pressure Advance value would then be:"
     )}
     {action_respond_info("")}
+    {action_respond_info("Assuming that first band_number is \0430")}

```

Comment:
What is this \0430 ?

Reply:
I should avoid that. \043 is octal value of unicode character # which makes in console "Assuming that first band_number is #0". If you consider the first band_number is 1 then the equation `PA = pa_start + ( pa_increment * band_number)` is wrong. 

Actually, you can consider two ways: Either you print the first band  with `Pressure_advance = pa_start` then first band_number = 0 or you print with   `Pressure_advance = pa_start  + pa_increment` then first band_number will be 1.

Reply:
It's ok, thank for the explanation about the unicode char, I was not aware this was possible !